README.rst

```diff
@@ -234,13 +234,33 @@ Configuration:
     # The maximum size of files to load into DataStore. In bytes. Default is 1 GB.
     ckanext.xloader.max_content_length = 1000000000
 
-    # To always use messytables to load data, instead of attempting a direct
-    # PostgreSQL COPY, set this to True. This more closely matches the
-    # DataPusher's behavior. It has the advantage that the column types
-    # are guessed. However it is more error prone, far slower and you can't run
-    # the CPU-intensive queue on a separate machine.
+    # By default, xloader will first try to add tabular data to the DataStore
+    # with a direct PostgreSQL COPY. This is relatively fast, but does not
+    # guess column types. If this fails, xloader falls back to a method more
+    # like DataPusher's behaviour. This has the advantage that the column types
+    # are guessed. However it is more error prone and far slower.
+    # To always skip the direct PostgreSQL COPY and use type guessing, set
+    # this option to True.
+    ckanext.xloader.use_type_guessing = False
+
+    # Deprecated: use ckanext.xloader.use_type_guessing instead.
     ckanext.xloader.just_load_with_messytables = False
 
+    # Whether ambiguous dates should be parsed day first. Defaults to False.
+    # If set to True, dates like '01.02.2022' will be parsed as day = 01,
+    # month = 02.
+    # NB: isoformat dates like '2022-01-02' will be parsed as YYYY-MM-DD, and
```

Comment:
Might be worth mentioning that non-ISO dates can have bad interactions between dayfirst and yearfirst. If both are enabled, then they will be parsed as yy-dd-mm which nowhere uses.